lib/zooniverse-react-components.css

```diff
@@ -122,3 +122,6 @@
   margin-right: 0.25em;
   white-space: nowrap;
 }
+.paginator .paginator-page-selector select {
```

Comment:
Ok, redid the commits to leave out the `lib` files generated by the build task :)